Cargo.toml

```diff
@@ -42,7 +42,7 @@ futures-core = "0.3.8"
 
 [dependencies.sqlx]
 version = "0.4.1"
-features = ["postgres"]
+features = ["any"]
```

Comment:
Is `"any"` a distinct feature flag for `sqlx`, or someting particular to rustc's feature support?

Reply:
@chrisdickinson "any" is a feature flag for `sqlx` that turns on the "Any" database driver. That more closely maps to what JDBC does and allows for configuring the db driver at runtime instead of compile-time. 

---

I would recommend specifying no db/type feature flags in Cargo.toml and recommending people to use tide-sqlx like this:

```rust
sqlx = { version = "0.4", features = [ "postgres" ] }
tide-sqlx = "0.1"
```

As features are additive, it'll apply inside tide-sqlx as well.